### PR TITLE
Home

### PR DESCRIPTION
**Pull request by louisaevola**
added a bunch of styling additions to home page (wip its still ugly lol)

### CODE REVIEW

**Review by mihirapatel — APPROVED** [review]

**Review by katiewu71 — APPROVED** [review]

File: src/components/Home/Home.js [hunk 1]
```diff
@@ -1,5 +1,79 @@
-import React from 'react';
+import React, {useEffect} from 'react';
+import SearchBar from '../SearchBar/SearchBar'
+import styles from './Home.scss'
+import { Container, Row, Col } from 'react-grid-system';
+import img from '../../assets/pictures/political.jpg'
+import Navigation from '../Navigation/Navigation'
+import NavigationBar from '../Navigation/NavigationBar';
+console.log(styles);
```
Comment (joe-tohdjojo) [inline src/components/Home/Home.js]: we don't want to have `console.log`'s in our production code so let's make sure to delete them before committing your code

File: src/components/Home/Home.scss [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+  flex: 1 1 auto;
+}
+
+.typewritertext {
+  box-sizing:border-box;
+  color: #000000;
+  font-family: 'Times New Roman'
+;
+  font-size: 70px;
+  // overflow: hidden; /* Ensures the content is not revealed until the animation */
```
Comment (joe-tohdjojo) [inline src/components/Home/Home.scss]: this line is commented out so we can probably delete it

File: src/index.html [hunk 3]
```diff
@@ -14,3 +15,9 @@
 </body>
 
 </html>
+
+<style>
+  body {
+    background-image:  url("assets/pictures/background.jpg");
+  }
+</style>
```
Comment (joe-tohdjojo) [inline src/index.html]: did you intend for this style block to be added here?

File: src/components/Navigation/NavigationBar.js [hunk 4]
```diff
@@ -0,0 +1,21 @@
...
+    <nav className={styles.navigation}>
+      <ul>
+        <li>
+          <Link to="/subjects/"><i className ="far fa-lightbulb"></i>   subjects </Link>
+        </li>
+        <li>
+          <Link to='/bills/'><i className ="far fa-file-alt"></i> bills</Link>
+        </li>
+        <li>
+          <Link to='/politicians/'><i className ="far fa-handshake"></i>  politicians</Link>
```
Comment (joe-tohdjojo) [inline src/components/Navigation/NavigationBar.js]: this is mainly a code style change. lets get rid of the space between `className` and `=`

```suggestion
          <Link to='/politicians/'><i className="far fa-handshake"></i>  politicians</Link>
```

File: src/components/Logo/Logo.js [hunk 5]
```diff
@@ -0,0 +1,12 @@
+import React from 'react';
+import {Link} from 'react-router-dom';
+import styles from './Logo.scss';
+import img from '../../assets/pictures/logo.png'
+
+export default function Logo() {
+  return (
+    <span>
+      <Link to="/"><img src={img} width="240px" height="80px" className = {styles.logo}/></Link>
```
Comment (joe-tohdjojo) [inline src/components/Logo/Logo.js]: Please remove the space between `className` and `=`

```suggestion
      <Link to="/"><img src={img} width="240px" height="80px" className={styles.logo}/></Link>
```

**Review by joe-tohdjojo — CHANGES_REQUESTED** [review]

File: src/index.html (hunk 3 above)
Comment (louisaevola) [reply src/index.html]: yes

Comment (joe-tohdjojo) [reply src/index.html]: typically, style blocks are placed in between the `<head>` tag. could you explain why they're outside of the `html` tags? this is not a typical way of using the `style` block so im curious why you've done it this way

**Review by joe-tohdjojo — APPROVED** [review]
Looks good to me

**Pull request merged**